bundle.md

```diff
@@ -14,13 +14,7 @@ This includes the following artifacts:
     This REQUIRED file MUST reside in the root of the bundle directory and MUST be named `config.json`.
     See [`config.json`](config.md) for more details.
 
-2. <a name="containerFormat02" />A directory representing the root filesystem of the container.
-    On Windows, for Windows Server containers, this directory is REQUIRED.
-    For Hyper-V containers, it MUST be omitted.
-
-    On all other platforms, this field is REQUIRED.
-
-    If set, this directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
+2. <a name="containerFormat02" />The directory referenced by [`root.path`](config.md#root), if that property is set in `config.json`.
```

Comment:
> Here, we want to express there is a directory which is referenced by config.md#root.

Yes, and it is *that directory* which is the bundle artifact.

> `The directory` will stress that directory is a specific directory, that's OK, but the base is everyone know which and what the directory is. I think there is not such base in context, it looks unreasonable.

But everyone should know which directory it is, since we say "referenced by `root.path`, if that property is set in `config.json`".  So the previous entry establishes the `config.json`artifact, and there can be only one file at that path.  And this entry establishes (when `root.path` is set in *that* configuration) *the* referenced directory as an artifact too.  Once you have the `confog.json`, there is no further choice about which directory you use for this artifact.